circuit_knitting/cutting/cutqc/dynamic_definition.py

```diff
@@ -52,7 +52,7 @@
 
 
 @deprecate_func(
-    removal_timeline="Circuit knitting toolbox 0.8.0 release",
+    removal_timeline="No sooner than CKT v0.8.0",
```

Comment:
```suggestion
    removal_timeline="no sooner than CKT v0.8.0",
```
since this ends up being part of a setence (look at the `deprecate_func` code).